README.md

```diff
@@ -76,6 +77,35 @@ EXAMPLES
 
 _See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v0.1.0/src/commands/autocomplete/index.ts)_
 
+## `chectl devfile:generate`
+
+generate and print a devfile to stdout given some Kubernetes resources and other Che workspaces features (project, language-support, commands etc...)
+
+```
+USAGE
+  $ chectl devfile:generate
+
+OPTIONS
+  -h, --help                 show CLI help
+  --command=command          Command to include in the workspace
+  --dockerImage=dockerImage  dockerImage component to include to the workspace
+  --editor=editor            Editor to choose. Currently supported editors: theia-next,theia-1.0.0
+  --git-repo=git-repo        Source code git repository to include in the workspace
+
+  --language=language        Add support for a particular language. Currently supported languages:
+                             java,typescript,go,python,c#
+
+  --name=name                Workspace name
```

Comment:
Currently, it's not true. It's a devfile name that will be reused for workspace name, and workspace name can be suffixed if workspace with such name already exists.

Reply:
ok it is the 'workspace desired name' so ... but the way it is dealing with already existing workspace name is a Che implementation choice: you could choose to fail if already exist. In the devfile, here, user would like to provide the workspace name even if in reality it would be suffixed.